samza-rest/src/test/java/org/apache/samza/monitor/TestLocalStoreMonitor.java

```diff
@@ -97,7 +97,7 @@ public void cleanUp()  {
     }
   }
 
-  @Test
+//  @Test
```

Comment:
👍 
Untested features are the most likely to break. Lets not let this stay disabled indefinitely.